src/DynamoCoreWpf/ViewModels/Watch3D/DynamoEffectsManager.cs

```diff
@@ -15,74 +16,48 @@ namespace Dynamo.Wpf.ViewModels.Watch3D
     /// </summary>
     public class DynamoEffectsManager : DefaultEffectsManager
     {
-        public DynamoEffectsManager(IRenderTechniquesManager renderTechniquesManager) : base(renderTechniquesManager) { }
+        public DynamoEffectsManager() : base() {
```

Comment:
another binary break - one option is to just keep this signature and ignore the use of the parameter, but I think obsoleting the method is cleaner.